pkg/email/parser.go

```diff
@@ -251,7 +251,16 @@ func extractCommand(body string) (cmd Command, str, args string) {
 		return
 	}
 	cmdPos += len(commandPrefix) + 1
-	for cmdPos < len(body) && (body[cmdPos] == ' ' || body[cmdPos] == '\t') {
+
+	isWhitespace := func(c byte) bool {
```

Comment:
There is `unicode.IsSpace`. It will also consume '\v', '\f', but I think it's find in this context.